patterns/project-layout.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Title: Project Layout
+ * Slug: twentytwentyfour/project-layout
+ * Categories: text
+ */
+
+?>
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"6vw","right":"6vw","bottom":"6vw","left":"6vw"}},"elements":{"link":{"color":{"text":"#fffffffc"}}},"color":{"background":"#181818","text":"#fffffffc"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-text-color has-background has-link-color" style="color:#fffffffc;background-color:#181818;padding-top:6vw;padding-right:6vw;padding-bottom:6vw;padding-left:6vw"><!-- wp:columns {"verticalAlignment":null,"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|50","left":"var:preset|spacing|50"}}}} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"verticalAlignment":"stretch","width":"50%","style":{"spacing":{"padding":{"right":"0%"}}}} -->
+<div class="wp-block-column is-vertically-aligned-stretch" style="padding-right:0%;flex-basis:50%"><!-- wp:group {"style":{"dimensions":{"minHeight":"100%"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"space-between","justifyContent":"stretch"}} -->
+<div class="wp-block-group" style="min-height:100%"><!-- wp:image {"id":106,"sizeSlug":"large","linkDestination":"none"} -->
+<figure class="wp-block-image size-large"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/tt4_portfolio_9-scaled-1.jpg " alt="" class="wp-image-106"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontSize":"0.8rem"}}} -->
+<p style="font-size:0.8rem"><?php echo esc_html__( "1. Queste basi mi hanno permesso di affrontare la sessione con un profondo apprezzamento per l'identità dell'hotel e una chiara visione della storia che volevo raccontare attraverso il mio obiettivo.", 'twentytwentyfour' ); ?></p>
```

Comment:
I think the text here shouldn't have the `1.` at the start

Reply:
The 1 is inside the new version on figma, shoud i delete it?

Reply:
Sorry, I didn't see the new style on figma, I had to refresh my browser. It's fine to keep, but let's change the function to use `esc_html_x` and explain what this text is referring to. I would love if we could use the image caption for this, but I'm not sure that we can while maintaining the design

Reply:
Yeah i thought about the caption but i think that without some custom css is impossible to have the same design of Figma.
Yeah, i'll use the esc_html_x thank u.

Reply:
Hope now is ok